explorer/app/config/model.ts

```diff
@@ -1,5 +1,34 @@
+import {
+  DetailResponseType,
+  DetailViewModel,
+  ListResponseType,
+  ListViewModel,
+} from "app/models";
 import { HeaderProps } from "../components/Header/Header";
 
+type ListTransformerFunction<T extends ListResponseType> = (
+  response: T
+) => ListViewModel;
+
+type DetailTransformerFunction<T extends DetailResponseType> = (
+  response: T
+) => DetailViewModel;
+
+type GetIdFunction<T extends DetailResponseType> = (detail: T) => string;
+
+export interface EntityConfig<
+  L extends ListResponseType = any,
```

Comment:
Two questions here:

1. Why do we need to default L and D to `any`?
2. Why is the linter not picking up the usage of `any` here? 

Thanks!

Reply:
I have attached a screenshot of the typings from a calling method. It would be great to see the actual types (rather than `any, any`).

<img width="529" alt="Screen Shot 2022-05-26 at 11 39 26 AM" src="https://user-images.githubusercontent.com/2446424/170554855-348fe423-9d2f-403c-ac3a-b616b8b21f66.png">

Reply:
I have attached a screenshot of the typings from a calling method. It would be great to see the actual types (rather than `any, any`).

<img width="529" alt="Screen Shot 2022-05-26 at 11 39 26 AM" src="https://user-images.githubusercontent.com/2446424/170554855-348fe423-9d2f-403c-ac3a-b616b8b21f66.png">